WordPress/Classes/ViewRelated/Blog/Site Picker/BlogList/SiteIconViewModel.swift

```diff
@@ -113,6 +118,7 @@ extension SiteIconViewModel {
 
 extension SiteIconViewModel {
     /// - parameter isBlavatar: A hint to skip the "is icon blavatar" check.
+    /// - parameter size: Size in pixels.
     static func makeReaderSiteIconURL(iconURL: String?, isBlavatar: Bool = false, siteID: Int?, size: CGSize) -> URL? {
```

Comment:
Nitpick: may as well rename the label `size:` -> `pixelSize:` 😄

Reply:
It's actually size in points 🤦 

This is how it's used:

```swift
  static func optimizedBlavatarURL(from path: String, imageSize: CGSize) -> URL? {
        let size = imageSize.scaled(by: UITraitCollection.current.displayScale)
        let query = String(format: "d=404&s=%d", Int(max(size.width, size.height)))
        return parseURL(path: path, query: query)
    }
```

I'm going to revert it.

Yeah, it's a low-level method. The app uses `SiteIconViewModel.Size` enum in most cases. It also ensures that you don't create too many different variants of the the same icon but off by a few points.